testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/mask.tests.js

```diff
@@ -374,6 +375,22 @@ QUnit.test("boundary value should correctly apply after second try to set overfl
 
 QUnit.module("format: text input", moduleConfig);
 
+QUnit.test("input should not be prevented in arabic locale", function(assert) {
+    var originalCulture = localization.locale();
+
+    try {
+        localization.locale("ar");
```

Comment:
What is the point to set Arabic locale if none localization engine is connected? Maybe it's better to mock `numberLocalization.format` method for this test?